packages/wonder-blocks-dropdown/index.js

```diff
@@ -1,6 +1,6 @@
-// // @flow
-// import ActionMenu from "./components/action-menu.js";
-// import SingleSelectMenu from "./components/single-select-menu.js";
-// import MultiSelectMenu from "./components/multi-select-menu.js";
-//
-// export {ActionMenu, SingleSelectMenu, MultiSelectMenu};
+// @flow
+import ActionMenu from "./components/action-menu.js";
+import SingleSelect from "./components/single-select.js";
+import MultiSelect from "./components/multi-select.js";
+
+export {ActionMenu, SingleSelect, MultiSelect};
```

Comment:
Yay!  Can you add a test for the exports of this file similar to what we do in https://github.com/Khan/wonder-blocks/blob/master/packages/wonder-blocks-typography/index.test.js?